main.go

```diff
@@ -42,3 +46,10 @@ func main() {
 	}
 
 }
+
+func formatBuiltWith(Tags string) string {
+	if len(Tags) > 0 {
+		return " built with: " + strings.Join(strings.Split(Tags, " "), ", ")
+	}
+	return ""
```

Comment:
Yes, is guess the special case is empty tags. Will update with both suggestions.